src/transform.ts

```diff
@@ -111,19 +122,19 @@ export interface WindowFieldDef {
   op: AggregateOp | WindowOnlyOp;
 
   /**
-   *  Parameter values for the window functions. Parameter value can be omitted for operations that do not accept a parameter.
+   *  Please refer to the operation/parameter table. Parameter values for the window functions. Parameter value can be omitted for operations that do not accept a parameter. The spec will be invalid if any function that requires a parameter is unspecified.
```

Comment:
> Please refer to the operation/parameter table. 

It's weird description to say refer first.  Better describe what it is first and say this later. 

> The spec will be invalid if any function that requires a parameter is unspecified.

This seems too extreme.  It should just be ignored without failing.